org.kde.ktuberling.json

```diff
@@ -9,6 +9,7 @@
         "--share=ipc",
         "--socket=cups",
         "--socket=x11",
```

Comment:
Please remove the socket=x11 line

Reply:
It's actually not necessary to remove this – if both `fallback-x11` and `x11` are specified then `x11` is ignored. But, OK>

Reply:
Updated the commit, and its commit message.